packages/playground/remote/src/lib/worker-thread.ts

```diff
@@ -52,9 +52,12 @@ if (
 	typeof navigator?.storage?.getDirectory !== 'undefined'
 ) {
 	virtualOpfsRoot = await navigator.storage.getDirectory();
-	virtualOpfsDir = await virtualOpfsRoot.getDirectoryHandle('wordpress', {
-		create: true,
-	});
+	virtualOpfsDir = await virtualOpfsRoot.getDirectoryHandle(
+		startupOptions.siteSlug,
```

Comment:
Let's prefix that with something or, even better, put it in /sites/ directory. This way we'll be able to safely use the rest of OPFS without risking finding a WordPress site in a random location.

Reply:
I added prefixes. If I moved sites to a subfolder the default WordPress site would be special or I had to be moved, so prefixes seemed cleaner.